lib/pbench/agent/base.py

```diff
@@ -104,3 +101,6 @@ def get_path(self, path):
             return pathlib.Path(path)
         else:
             return path
+
+    def tool_group_dir(self, group):
+        return pathlib.Path(self.pbench_run, f"tools-v1-{group}")
```

Comment:
Seems like this could be simplified to be `return self.pbench_run / f"tools-v1-{group}"`.  Worth doing?